### DISCUSSION TITLE
Standardize school-timetabling without UI quickstart

### DISCUSSION THREAD

**Discussion by zepfred** [TimefoldAI/timefold-solver#487] [General]
The [school timetabling without UI](https://github.com/TimefoldAI/timefold-quickstarts/blob/stable/use-cases/school-timetabling/README.adoc) quickstart uses a simplified data model, which doesn't use database and doesn't allow to add/remove/edit any items. It also uses a slightly different UI because of that.

The same changes need to be made to the school timetabling without UI quickstart. The quickstarts should be identical to the Quarkus quickstart, except for the unnecessary dependencies. (Same names of field, classes, same data sets etc.)

**Comment by triceo** [comment]
Thanks, @zepfred!
I created an issue and moved it to the quickstarts repo. 
We can continue there:

https://github.com/TimefoldAI/timefold-quickstarts/issues/192
